### DISCUSSION TITLE
Pregnancy/Fetus model Aging

### DISCUSSION THREAD

**Discussion by ReneGeci** [Open-Systems-Pharmacology/Forum#1663] [Questions & Problems] [answered]
Hello,

I was trying to use the Dallmann 2017 Pregnancy Model to simulate daily administration to a woman over an entire 9 months pregnancy. And I was initially assuming that it should be possible to dynamically take into account the physiological/anatomical pregnancy related ageing/maturation processes in the simulation. However, this seems not to be possible. When creating a Pregnant PBPK Model in MoBi as outlined in the Dallmann 2018 Tutorial, the parameters in the pregnant woman spatial structure are static, so do not include dynamic ageing.

Can somebody confirm that what I intended is not possible?
And finally, isnt the information required for this present in the PK Sim Population database? So shouldnt it in principle be possible to do include these physiol/anatomical ageing/maturation processes?

**Comment by AndreDlm** [comment, marked as answer]
No, unfortunately a "growing" pregnancy population is currently not possible, although all information is in the database.

I created a feature request for that a long time ago, if I find it again, I will post it here so that it gets more support from the community ;-)

**Reply by Yuri05** [reply]
this? https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1458

**Reply by StephanSchaller** [reply]
Not sure this covers the fetus?
So for the fetus we would have to map preterm data to the fetus model to allow ageing? Also the placenta physiology will change.

So, where is all required data? We might take this up next year, then. (Assign Rob if you like ;-))

**Reply by AndreDlm** [reply]
You are right, some parameters need further consideration. The fetal weight is one example where the weight at a given gestational age should ideally match that of (pre)term neonates in the software. The placenta obviously changes structurally during the germinal and embryonic phase, but in a first step you could focus on fetal devlopment (second + third trimester) which would simplify a lot of things.

**Comment by ReneGeci** [comment]
@AndreDlm, thanks for the quick answer!
that would really be a nice feature, especially since the data is already there...
